src/Setup/DevDivVsix/ServiceHubConfig/Roslyn.VisualStudio.Setup.ServiceHub.csproj

```diff
@@ -0,0 +1,58 @@
+﻿<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net472</TargetFramework>
+    <RoslynProjectType>Custom</RoslynProjectType>
+
+    <!-- VS Insertion -->
+    <TargetVsixContainerName>Roslyn.VisualStudio.Setup.ServiceHub.vsix</TargetVsixContainerName>
+    <VisualStudioInsertionComponent>Microsoft.CodeAnalysis.LanguageServices</VisualStudioInsertionComponent>
+
+    <ExtensionInstallationRoot>CommonExtensions</ExtensionInstallationRoot>
```

Comment:
@tmat another duplicated string. what would be right way to share this with Roslyn.VS.Setup.csproj?